src/settings/Settings.tsx

```diff
@@ -101,6 +101,10 @@ export enum LabGroup {
     Developer,
 }
 
+export const FEATURES = {
+    VOICE_BROADCAST: "voice_broadcast",
```

Comment:
maybe should still use the `feature_` prefix for the value to match the existing feature settings? also could be nice to use an enum

Reply:
++ for real enum, also it think as per the new code style this shouldn't be all caps